hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/SubProcessHapiTestNode.java

```diff
@@ -92,11 +164,16 @@ public void start() {
             builder.command(
                             javaCmd,
                             "-Dfile.encoding=UTF-8",
+                            // You can attach to any node. Node 0 at 5005, node 1 at 5006, etc. But if you need the
+                            // node to stop at startup, you can change the below line so nodeId == the node you want
+                            // to suspend at startup and the first "n" to "y".
                             "-agentlib:jdwp=transport=dt_socket,server=y,suspend=" + (nodeId == 0 ? "n" : "n")
                                     + ",address=*:" + (5005 + nodeId),
-                            "-Dhedera.workflows.enabled=true",
                             "-classpath",
                             classPath,
+                            "-Dfile.encoding=utf-8",
+                            "-Dhedera.workflows.enabled=true",
+                            "-Dprometheus.endpointPortNumber=" + (10000 + nodeId),
                             "com.hedera.node.app.ServicesMain",
```

Comment:
Suggested replacement for lines 166-177 (github doesn't allow change suggestions if any line in the section is deleted)...
```java
                            // You can attach to any node. Node 0 at 5005, node 1 at 5006, etc. But if you need the
                            // node to stop at startup, you can change the below line so nodeId == the node you want
                            // to suspend at startup and the first "n" to "y".
                            "-agentlib:jdwp=transport=dt_socket,server=y,suspend=" + (nodeId == 0 ? "n" : "n")
                                    + ",address=*:" + (5005 + nodeId),
                            "-classpath",
                            classPath,
                            "-Dfile.encoding=UTF-8",
                            "-Dhedera.workflows.enabled=true",
                            "-Dprometheus.endpointPortNumber=" + (10000 + nodeId),
                            "com.hedera.node.app.ServicesMain",

```